packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/BucketMap.ts

```diff
@@ -6,7 +6,11 @@
  * and also tracks the total count of objects stored.
  */
 export class BucketMap<T = any> extends Map<string, T[]> {
```

Comment:
Should this be pulled up out of the `FileSystemGraphObjectStore` directory? Seems it's part of the `storage` module, used by the `FileSystemGraphObjectStore`, but not dependent/exclusive to it.